<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="welcomeWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>371</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="2" colspan="2">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font-size:16px;margin:0 auto</string>
       </property>
       <property name="text">
        <string>     Commencez Ã  jouer au Shifumi !</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="roundLabel">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:12px; margin:0 auto;</string>
       </property>
       <property name="text">
        <string>      Entrez le nombre de manches gagnantes</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="roundInput">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="pseudoLabel">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:12px;margin:0 auto; </string>
       </property>
       <property name="text">
        <string>                         Entrez votre pseudo</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="pseudoInput">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="playButton">
       <property name="styleSheet">
        <string notr="true"> background-color: #e7e7e7; /* Green */
  border: none;
  color: dark;
  padding: 7px 16px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 10px;
 border-radius: 32px</string>
       </property>
       <property name="text">
        <string>Jouer !</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gameWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="rockButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #BC8F8F; /* Green */
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
 border-radius: 32px</string>
     </property>
     <property name="text">
      <string>Pierre</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paperButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>200</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #4CAF50; /* Green */
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
 border-radius: 32px</string>
     </property>
     <property name="text">
      <string>Feuille</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scissorsButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>200</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #555555; /* Green */
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
 border-radius: 32px</string>
     </property>
     <property name="text">
      <string>Ciseaux</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="endWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="endMessage">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>240</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Entrer les round max gagnant</string>
     </property>
    </widget>
    <widget class="QPushButton" name="restartButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>270</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #e7e7e7; /* Green */
  border: none;
  color: dark;
  padding: 7px 16px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 10px;
 border-radius: 32px</string>
     </property>
     <property name="text">
      <string>Rejouer !</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="resultWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="resultLabel">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>291</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #fff; /* Green */
  border: none;
  color: dark;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
 border-radius: 32p;x</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="iaChoice">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>220</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #fff; /* Green */
  border: none;
  color: dark;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
 border-radius: 32p;x</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <zorder>gameWidget</zorder>
   <zorder>endWidget</zorder>
   <zorder>welcomeWidget</zorder>
   <zorder>resultWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>749</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rockButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>rock()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>875</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>paperButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>paper()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>686</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scissorsButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>scissors()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>627</x>
     <y>634</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restartButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startGame()</slot>
  <slot>rock()</slot>
  <slot>paper()</slot>
  <slot>scissors()</slot>
 </slots>
</ui>
